.github/release.yml

```diff
@@ -8,28 +8,26 @@ changelog:
   categories:
     - title: Security Fixes
       labels:
-        - C-security
+        - type:security
 
     - title: New Features
       labels:
-        - C-new-feature
-
-    - title: Feature Enhancements
-      labels:
-        - C-enhancement
+        - type:new-feature
 
     - title: Usability and Accessibility
       labels:
-        - "A-*"
+        - type:accessibility
+        - type:usability
+        - type:visual-clarity
 
     - title: Bug Fixes
       labels:
-        - bug
+        - type:bug
 
     - title: Documentation
       labels:
-        - documentation
+        - type:documentation
 
     - title: Other Changes
       labels:
-        - "*"
+        - "type:*"
```

Comment:
I think I would keep `*` as that means all unlabeled PRs also end up in the changelog. Otherwise it's probably fairly easy to forget some.

Reply:
I thought all PRs would end up in the changelog anyway. :thinking: 
These categories are for the release notes, aren't they usually just a summary of the most important changes?

Or am I wrong?

Reply:
Uhm  I'm not sure. Like for Studio there is only one "thing" and not two separate "changelog" and "release notes". 

So these automatic release notes are generated as draft release right? And a human checks the release notes before actual release. I'm just trying to avoid that we forget to label a noteworthy PR, and then just look at the release notes (where it is not listed) and then we forget mentioning it at all. 
So the idea is to make Github generate release notes where all PRs that are not tagged with the labels above end up in the "other changes" section. Then a human can decide whether they are boring changes or whether we just forgot to assign a label.
Does this make sense?

Reply:
> I'm just trying to avoid that we forget to label a noteworthy PR, and then just look at the release notes (where it is not listed) and then we forget mentioning it at all.

That definitely makes sense. Then I agree with you there.
Will change it.